### DISCUSSION TITLE
Senderapport ausschalten

### DISCUSSION THREAD

**Discussion by schweizerindeutsch** [PHPMailer/PHPMailer#2277] [General]
Wie kann ich den Senderapport ausschalten, dieser stört mein System beim weitergeben der Variablen.

Dieser Rapport meine ich
2021-03-03 12:56:13 SERVER -> CLIENT: 220 server.deESMTP Postfix (Debian/GNU)
2021-03-03 12:56:13 CLIENT -> SERVER:
usw.

Suche die Funktion zum deaktivieren der Statusnachrichten

**Comment by Synchro** [comment]
$mail->SMTPDebug = 0;

**Reply by schweizerindeutsch** [reply]
Perfekt herzlichen Dank!!
